nmclt-standalone.rst

```diff
@@ -0,0 +1,89 @@
+===============================================
+NMCTL - how to use netmaker from command line
+===============================================
+
+a brief guide to using netmaker from the command line (without the UI)
+
+******************
+Assumptions
+******************
+
+1. using bash shell
+2. netclient, nmctl and jq have been installed
+3. netmaker server has been set up at example.com without UI  (netmaker-ui section of docker-compose.yml has been deleted)
+
+
+***********************
+Set username/password/masterkey
+***********************
+*masterkey* must match the masterkey on server
+.. code-block::
+    
+        export USER=admin
+        export PASSWORD=somelongpassword
+
+***********************
+Set masterkey
+***********************
+*masterkey* must match the masterkey on server
+.. code-block::
+        export MASTERKEY=masterkey
+
+******************
+Set base domain
+******************
+.. code-block::
+        
+            export NN_DOMAIN=example.com
+
+******************
+Set Context
+******************
+.. code-block::
+            
+            nmctl context set commandline --endpoint https://api.$NM_DOMAIN --master_key $MASTERKEY
+            nmctl context use commandline
```

Comment:
why use master_key?
docs should suggest user to use their username and password, rather than master key

Reply:
because user hasn't been created yet so have to use masterkey

Reply:
first user (super admin) can be created without any auth

Reply:
if all users are expected to use master key what's point of RBAC on server then ?
master key's main intention  is only to recover the server, if when say super admin forgets his password for example

Reply:
cannot set context without either masterkey or username/password .... if set to username/password user create will fail as user does not exist.

added a step to reset the context

Reply:
yes, the idea is superadmin is the one who will create other users, superadmin is the first user who will be created on the server, so don't have to use masterkey

Reply:
either  masterkey or username/password is required to set context

Reply:
users will have their usernames and password to set context, all the users are not expected to have master key with them

Reply:
added clarification that initial admin must be created by superAdmin